templates/poster.html

```diff
@@ -71,14 +71,45 @@ <h3 class="card-subtitle mb-2 text-muted text-center">
   </div>
 </div>
 
-<!-- Presentation Embed -->
-{{ components.section("Paper") }}
-
 <h5 style="color: red;">
   Add content for posters. This could be a video, embedded pdf, chat room ....
 </h5>
 
-    <h4> Example Poster (using pdf.js): </h4>
+<!-- Chat -->
+{{ components.section("Example Chat") }}
+
+<div class="col-md-12 col-xs-12 p-2">
+  <div id="gitter" class="slp">
+    <center>
+      <iframe frameborder="0" src="http://{{config.chat_server}}/channel/paper_{{paper.id}}?layout=embedded" height="700px" width="100%" ></iframe>
+    </center>
+  </div>
+</div>
+
+<!-- Slides Live-->
+
+{{ components.section("Example SlidesLive") }}
+
+<div class="col-md-12 col-xs-12 my-auto p-2" >
+  <div id="presentation-embed-38922815" class="slp my-auto"></div>
```

Comment:
Is it possible to change `38922815` a config variable, e.g., `{{paper.presentation_id | default(config.default_presentation_id) }}`?
(Same for Line 97 and Line 98.)

Reply:
Good idea.